services/dumper.js

```diff
@@ -137,18 +145,42 @@ function Dumper(config) {
     return stringUtils.camelCase(stringUtils.transformToSafeString(table));
   }
 
+  function getSafeDefaultValue(field) {
+    // NOTICE: in case of SQL dialect, ensure default value is directly usable in template
+    //         as a JS value.
+    let safeDefaultValue = field.defaultValue;
+    if (config.dbDialect !== 'mongodb') {
+      if (typeof safeDefaultValue === 'object' && safeDefaultValue instanceof Sequelize.Utils.Literal) {
+        safeDefaultValue = `Sequelize.literal('${safeDefaultValue.val}')`;
+      } else if (
+        !_.isNil(safeDefaultValue)
+        && _.some(
+          DEFAULT_VALUE_TYPES_TO_STRINGIFY,
+          // NOTICE: Uses `startsWith` as composite types may vary (eg: `ARRAY(DataTypes.INTEGER)`)
+          (dataType) => _.startsWith(field.type, dataType),
```

Comment:
[Thinking] This could be considered not reliable to analyze the beginning of the string (i.e `DATEONLY` vs `DATE`). However, in that particular case, we could argue it's not a problem! (but it could be later since the contract is hidden).

Reply:
Agreed, but as you said, this should not be an issue here. Starting with `DATE` means "I am a date value", and the code says "All date values should be stringified". We should be safe for now.

Added `DATE` to the list of type to quote btw. Thanks!